sbnanaobj/StandardRecord/SROpFlash.h

```diff
@@ -0,0 +1,48 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SROPFLASH.h
+// \brief   Adaptation of recob::OpFlash (https://nusoft.fnal.gov/larsoft/doxsvn/html/classrecob_1_1OpFlash.html). This SR code copied/based on other SR objects.
+// \author  jsmedley@fnal.gov
+////////////////////////////////////////////////////////////////////////
+#ifndef SROPFLASH_H
+#define SROPFLASH_H
+
+#include "sbnanaobj/StandardRecord/SRConstants.h"
+#include <climits>
+#include <vector>
+
+namespace caf
+{
+  /// OpFlash
+  class SROpFlash
+    {
+    public:
+      SROpFlash();
+
+      bool  OnBeamTime  { false             }; //!< Is this in time with beam?
+      float Time        { kSignalingNaN     }; //!< Time on trigger time scale [us].
+      float TimeWidth   { kSignalingNaN     }; //!< Width of the flash in time [us].
+      float TimeMean    { kSignalingNaN     }; //!< Mean time of hits [us].
+      float TimeSD      { kSignalingNaN     }; //!< Standard deviation of hit times [us].
+      float FirstTime   { kSignalingNaN     }; //!< Time of first hit [us].
+      float TotalPE     { kSignalingNaN     }; //!< Total number of PE across all PMTs.
+      float FastToTotal { kSignalingNaN     }; //!< Fast to total light ratio.
+      float YCenter     { kSignalingNaN     }; //!< Geometric center in y [cm].
+      float YWidth      { kSignalingNaN     }; //!< Geometric width in y [cm].
+      float ZCenter     { kSignalingNaN     }; //!< Geometric center in z [cm].
+      float ZWidth      { kSignalingNaN     }; //!< Geometric width in z [cm].
+      float XCenter     { kSignalingNaN     }; //!< Geometric center in x [cm].
+      float XWidth      { kSignalingNaN     }; //!< Geometric width in x [cm].
+      int   Cryo        { kUninitializedInt }; //!< 0 for SBND/ICARUS East, 1 for ICARUS West; NOT IN THE RECOB
+      int   FirstPMT    { kUninitializedInt }; //!< Channel number of first hit
```

Comment:
The CAF variable naming scheme is all-lowercase (to a first approximation). I think all of these would look fine just directly lowercased from their current names.

I think it would make sense to use `SRVector3D` for the center and width fields though.

And I think `fastfrac` or `fracfast` makes more sense to me than `FastToTotal`. If you're directly copying a name in the recob object then there's more of an argument for keeping it.